editor/src/core/shared/github/endpoints.ts

```diff
@@ -18,6 +18,15 @@ export const GithubEndpoints = {
       'branch',
       branchName,
     ),
+  defaultBranchContents: (githubRepo: GithubRepo) =>
+    urljoin(
+      UTOPIA_BACKEND,
+      'github',
+      'branches',
+      githubRepo.owner,
+      githubRepo.repository,
+      'default-branch',
```

Comment:
shall we stick wi GH's (unchanged) [default](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-branches#about-the-default-branch) here? (`main`)